### PR TITLE
Wrongly linked module

### PR DESCRIPTION
**Pull request by tomlarse**
Proposed fix for #374

I also updated some formatting to make it look nicer.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/configure-skype-for-business-for-hybrid-modern-authentication.md [hunk 1]
```diff
@@ -113,38 +115,43 @@ Now you'll need to run commands to add the URLs (collected earlier) as Service P
 1. First, connect to AAD with [these instructions](https://docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-1.0).
     
 2. Run this command, on-premises, to get a list of SFB web service URLs.
+
+   Notice that the AppPrincipalId begins with `00000004`. This corresponds to Skype for Business Online.
```
Comment (JohanFreelancer9) [inline Enterprise/configure-skype-for-business-for-hybrid-modern-authentication.md]: **Note** that the AppPrincipalId begins with `00000004`. This corresponds to Skype for Business Online.

Please do change as indicated in bold.

Comment (tomlarse) [reply Enterprise/configure-skype-for-business-for-hybrid-modern-authentication.md]: Hi

I've changed the wording as suggested

**Comment by mypil** [comment]
@MSFTTracyP - can you please assist to check, approve and merge the content for this PR based on issue #374?

Thank you.

**Comment by mypil** [comment]
@MSFTTracyP - We would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #374?

Thank you.

**Comment by mypil** [comment]
@MSFTTracyP - We would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #374?

Thank you.

**Comment by mypil** [comment]
@MSFTTracyP - We would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #374? Thank you.

cc: @LauraWi

**Pull request merged**